microsoft-edge/webview2/how-to/winrt-from-js.md

```diff
@@ -0,0 +1,336 @@
+
+
+<!-- ====================================================================== -->
+## Step 1: Clone the repo and build the WebView2 UWP sample
+
+1. If Visual Studio 2015 or later is not already installed, in a separate window or tab, see [Install Visual Studio](../how-to/machine-setup.md#install-visual-studio) in _Set up your Dev environment for WebView2_.  Follow the steps in that section, and then return to this page and continue the steps below.
+
+1. If a preview channel of Microsoft Edge (Beta, Dev, or Canary) is not already installed, in a separate window or tab, see [Install a preview channel of Microsoft Edge](../how-to/machine-setup.md#install-a-preview-channel-of-microsoft-edge) in _Set up your Dev environment for WebView2_.  Follow the steps in that section, and then return to this page and continue the steps below.
+
+   If you have your own app code base already, you can open that project in Visual Studio, instead of the following sample:
+
+1. If not done already, download or clone the `WebView2Samples` repo to your local drive.  In a separate window or tab, see [Download the WebView2Samples repo](../how-to/machine-setup.md#download-the-webview2samples-repo) in _Set up your Dev environment for WebView2_.  Follow the steps in that section, and then return to this page and continue below.
+
+1. On your local drive, open the `.sln` file in Visual Studio, in the directory:
+
+   *  `<your-repos-directory>/WebView2Samples/SampleApps/webview2_sample_uwp/webview2_sample_uwp.sln`
+
+   or:
+
+   *  `<your-repos-directory>/WebView2Samples-master/SampleApps/webview2_sample_uwp/webview2_sample_uwp.sln`
+
+   ![Adding a new project for the WebView2 WinRT JS Projection tool.](winrt-from-js-images/add-new-project-for-tool.png)
+
+1. In Visual Studio, select **Debug** > **Start Debugging**.  This first builds the **webview2_sample_uwp** project, and then runs the baseline version of the project.  The **WebView2 WinUI 2 UWP Sample** window opens.
+
+   ![The WebView2 WinUI 2 UWP Sample window.](winrt-from-js-images/webview2-winui-2-uwp-sample-app-window.png)
+
+1. Close the **WebView2 WinUI 2 UWP Sample** window.
+
+In case you need more information, see detailed steps in these pages, and then continue below:
+* [WebView2 sample: WinUI 2 (UWP) browser app](../samples/webview2_sample_uwp.md) - steps to open, compile, and run the sample app.
+* [Get started with WebView2 in WinUI 2 (UWP) apps (public preview)](../get-started/winui2.md) - setting up a basic WebView2 app.
+* [GitHub > WebView2Samples repo > webview2_sample_uwp](https://github.com/MicrosoftEdge/WebView2Samples/tree/master/SampleApps/webview2_sample_uwp)
+* [Set up your Dev environment for WebView2](machine-setup.md) - details about setting up prerequisites.
+
+
+<!-- =============================================== -->
+## Step 2. Add a WinRTAdapter project to contain the WebView2 WinRT JS Projection tool
+
+<!-- delete paragraph? why would you want to do this? -->
```

Comment:
@david-risney  @jasonstephen15 - delete paragraph? why would you want to do this?